src/UsersModule/Components/RoomDetails/RoomDetails.tsx

```diff
@@ -1,7 +1,397 @@
+  
+      console.log(response)
+    } catch (error) {
+      console.error("Somthing went wrong", error);
+    }
+    
+  }
+  
+  useEffect(() => {
+    setIsLoading(true);
+   getRoomDetail();
+  },[]);
+  function srcset(images: [], size: number, rows = 1, cols = 1) {
+    return {
+      src: `${images}?w=${size * cols}&h=${size * rows}&fit=crop&auto=format`,
+      srcSet: `${images}?w=${size * cols}&h=${
+        size * rows
+      }&fit=crop&auto=format&dpr=2 2x`,
+    };
+  }
   return (
-    <div>RoomDetails</div>
+    <>
+       <Box sx={{minHeight:"30vw",width:"90%" ,margin:"auto"}}>
+       { !isLoading ? (
+      <Loading/>
+    ) :<>
+       <Typography variant="h4" textAlign={'center'} >Village Angaa</Typography>
+       <Typography variant="body1" style={{ marginRight: "5px" }}>
+                <Link
+                  to="/"
+                  style={{ color: "#B0B0B0", textDecoration: "none" }}
+                >
+                  Home
+                </Link>
+              </Typography>
+       <Grid container spacing={2} p={5}>
+            {roomInfo?.images?.map((image, index) => (
+                <Grid item key={index} xs={12} >
+                    <Card>
+                        <CardMedia
+                            component="img"
+                            alt={image.alt}
+                           height={350}
+                            image={image} // image.url should be the URL of the image
+                         
+                        />
+                    </Card>
+                </Grid>
+            ))}
+        </Grid>
+     
+       <Grid container   rowSpacing={1} columnSpacing={{ xs: 1, sm: 2, md: 3 }}>
+        <Grid item xs={6} p={2}>
+        <Typography  color="text.secondary" sx={{ opacity: 0.6 }} variant="p">Minimal techno is a minimalist subgenre of techno music. It is characterized by a stripped-down aesthetic that exploits the use of repetition and understated development. Minimal techno is thought to have been originally developed in the early 1990s by Detroit-based producers Robert Hood and Daniel Bell.
+        Such trends saw the demise of the soul-infused techno that typified the original Detroit sound. Robert Hood has noted that he and Daniel Bell both realized something was missing from techno in the post-rave era.
+        Such trends saw the demise of the soul-infused techno that typified the original Detroit sound. Robert Hood has noted that he and Daniel Bell both realized something was missing from techno in the post-rave era.
+        </Typography>
+        <Box
+        sx={{padding:"5px",
+          textAlign:"center"
+        }}>
+        <Box
+                  sx={{
+                    marginTop: "2rem",
+                    display: "flex",
+                    justifyContent: "center",
+                  }}
+                >
+                  <Box>
+                    <BedIcon
+                      style={{
+                        color: "#152C5B",
```

Comment:
As I mentioned before, we shouldn't be using fixed colors and units, instead we should be using them from the design system.